google-cloud-bigtable/pom.xml

```diff
@@ -96,10 +87,6 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-api</artifactId>
     </dependency>
-    <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-auth</artifactId>
-    </dependency>
```

Comment:
Can we keep this dependency? Although we don't reference it, I think its important to keep consistent with all of the other grpc deps we depend on